src/components/Card/index.tsx

```diff
@@ -41,25 +40,35 @@ const BaseCard = ({
 	const Rows =
 		rows &&
 		map(rows, (row, index: number) => (
-			<Box key={index} fontSize={2}>
+			<Flex
+				key={index}
+				fontSize={2}
+				flexDirection="column"
+				flexWrap="wrap"
+				width="100%"
+			>
 				{index > 0 && <Divider />}
 				{row}
-			</Box>
+			</Flex>
 		));
 
 	const Body = children && (
-		<Box fontSize={2} height="100%">
+		<Flex
+			fontSize={2}
+			height="100%"
+			flexDirection="column"
+			flexWrap="wrap"
+			width="100%"
```

Comment:
Should we instead of `width="100%"` elements, apply an `alingItems="stretch"` on the Wrapper on line 68?